roles/cinder-common/defaults/main.yml

```diff
@@ -51,3 +54,9 @@ cinder:
     debug: False
     verbose: True
   cafile: "{{ ssl.cafile|default('/etc/ssl/certs/ca-certificates.crt') }}"
+  ceph:
+    rbd_user: cinder
+    rbd_ceph_conf: /etc/ceph/ceph.conf
+    rbd_flatten_volume_from_snapshot: false
+    rbd_max_clone_depth: 5
+    glance_api_version: 2
```

Comment:
Yeah, glance needs to know the rbd_ceph_conf too, although it isn't currently templated, just hard coded. Oops.